rust/src/lib.rs

```diff
@@ -297,12 +373,34 @@ pub unsafe extern "C" fn zcashlc_derive_extended_spending_keys(
     unwrap_exc_or_null(res)
 }
 
+/// A struct that contains an account identifier along with a pointer to the string encoding
+/// of a [`UnifiedFullViewingKey`] value.
+///
+/// # Safety
+///
+/// - `encoding` must be non-null and must point to a null-terminated UTF-8 string.
+///   and it must be aligned to the size of a u32.
```

Comment:
Why must the pointer be u32-aligned? That isn't usually the case for strings.

Reply:
Or do you mean the `encoding` field itself is 32-bit aligned? That is implied by `repr(C)` and alignment of the overall struct on most platforms.